modules/core-services/services/application/appliation.go

```diff
@@ -769,12 +778,11 @@ func (a *Application) UpdatePublishItemRelations(request *apistructs.UpdateAppPu
 func (a *Application) PipelineCmsConfigRequest(request *apistructs.UpdateAppPublishItemRelationRequest) error {
 	for workspace, mk := range request.AKAIMap {
 		// bundle req
-		if err := a.bdl.CreateOrUpdatePipelineCmsNsConfigs(a.BuildItemMonitorPipelineCmsNs(request.AppID, workspace.String()),
-			apistructs.PipelineCmsUpdateConfigsRequest{
-				KVs: map[string]apistructs.PipelineCmsConfigValue{"AI": {Value: mk.AI},
-					"AK": {Value: mk.AK}},
-				PipelineSource: apistructs.PipelineSourceDice,
-			}); err != nil {
+		if _, err := a.cms.UpdateCmsNsConfigs(context.Background(), &cmspb.CmsNsConfigsUpdateRequest{
+			Ns:             a.BuildItemMonitorPipelineCmsNs(request.AppID, workspace.String()),
+			PipelineSource: apistructs.PipelineSourceDice.String(),
+			Kvs:            map[string]*cmspb.PipelineCmsConfigValue{"AI": {Value: mk.AI}, "AK": {Value: mk.AK}},
+		}); err != nil {
```

Comment:
explain that even struct name and params changed, the reason why

Reply:
https://developers.google.com/protocol-buffers/docs/style#message_and_field_names

Use underscore_separated_names for field names. So I use `kvs` as field name and generated var name is Kvs, and there is no option to change this generation behaviour.

Now I use `KVs` as field name and set `json_name` option to `kvs`.